fennel/testing/test_utils.py

```diff
@@ -167,6 +257,42 @@ def proto_to_dtype(proto_dtype) -> str:
         return str(proto_dtype)
 
 
+def cast_df_to_arrow_dtype(
+    df: pd.DataFrame, fields: List[Field]
+) -> pd.DataFrame:
+    """
+    This helper function casts Pandas DatFrame columns to arrow dtype using list of Field proto.
+    """
+    for f in fields:
+        try:
+            series = cast_col_to_arrow_dtype(df[f.name], f.dtype)
+            series.name = f.name
+            df[f.name] = series
+        except Exception as e:
+            raise ValueError(
+                f"Failed to cast column `{f.name}` of type `{proto_to_dtype(f.dtype)}` to arrow dtype. Error: {e}"
+            )
+    return df
+
+
+def cast_df_to_pandas_dtype(
+    df: pd.DataFrame, fields: List[Field]
+) -> pd.DataFrame:
+    """
+    This helper function casts Pandas DatFrame columns to arrow dtype using list of Field proto.
```

Comment:
Both function have the same description. This one is incorrect i believe ?

Reply:
yup changed the description